src/goto-instrument/contracts/contracts.h

```diff
@@ -39,19 +40,10 @@ Date: February 2016
   " --replace-call-with-contract <fun>\n"                                      \
   "                              replace calls to fun with fun's contract\n"
 
-#define FLAG_REPLACE_ALL_CALLS "replace-all-calls-with-contracts"
-#define HELP_REPLACE_ALL_CALLS                                                 \
-  " --replace-all-calls-with-contracts\n"                                      \
-  "                              as above for all functions with a contract\n"
-
 #define FLAG_ENFORCE_CONTRACT "enforce-contract"
 #define HELP_ENFORCE_CONTRACT                                                  \
   " --enforce-contract <fun>     wrap fun with an assertion of its contract\n"
 
-#define FLAG_ENFORCE_ALL_CONTRACTS "enforce-all-contracts"
```

Comment:
I am a little concerned by the change in functionality.  I realise that it would be good to have the extra flexibility but having to give a command-line argument for every function is quite a lot.  Elsewhere I have seen regexes used for this kind of thing.

Reply:
We we're using contracts now in a context where an user enforces one contract at a time, makes sure it is correct and then uses it during replacement. In the future, we still want to have an option where we can simply apply contracts whenever they are available, but for now we want to force users to specify which contracts must be enforced or replaced.